cfn/templates/ecs-task.template.yml

```diff
@@ -154,7 +154,7 @@ Resources:
     Type: AWS::IAM::Role
     Properties:
       Description: A role for event to execute Klaxon ECS task
-      RoleName: schedule-ecs-task-role
+      RoleName: {{ events_role }}
```

Comment:
Going to suggest something slightly different here. 

Since we're always going to need to create this role, could we derive a unique name based on some other stuff in this template? That way we don't have to manually define something in the config. 

For example, since the name of this role is pretty inconsequential, we could swap this for something like:

```
RoleName: {{ family }}-role
```

what do you think?